app/src/main/res/values-fa/strings.xml

```diff
@@ -0,0 +1,173 @@
+<resources>
+    <string name="app_name">یوزرلند</string>
+    <string name="welcome">به یوزرلند خوش آمدید!</string>
+    <string name="permission_request_code">1000</string>
+
+    <!-- Labels -->
+    <string name="apps">کاره‌ها</string>
+    <string name="sessions">جلسات</string>
+    <string name="edit_session">ویرایش جلسه</string>
+    <string name="filesystems">فایل‌سیستم‌ها</string>
+    <string name="edit_filesystem">ویرایش فایل‌سیستم</string>
+    <string name="settings">تنظیمات</string>
+    <string name="help">راهنما</string>
+    <string name="toc">شرایط و ضوابط</string>
+
+    <!-- Buttons -->
+    <string name="button_continue">ادامه</string>
+
+    <!-- Menus -->
+    <string name="app_info">اطلاعات کاره</string>
+    <string name="stop_app">توقف کاره</string>
+    <string name="kill_service">Kill Service</string>
```

Comment:
Looks like `Kill Service` still requires translation.

Reply:
I left the translation to be done late since it would make no sense if I translate them literally. I need to figure our how else I can translate such terms. I think it can be done later.

Reply:
Some other phrases in English that communicate the same idea: 
- `Stop Session`
- `Kill Session`
- `End Service`
- `End Session`
- `Terminate Session`
- `Terminate Service`

and other similar phrasing. Let me know if that's helpful or if you'd like more suggestions.